### PR TITLE
Add section about downgrading packages.

### PR DESCRIPTION
**Pull request by flexibeast**
Having recently had to downgrade `dhcpcd`, i thought it might be useful to port the wiki information about downgrading across to the handbook. :-)

### CODE REVIEW

File: src/xbps/packages/downgrading.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Downgrading to a specific package version
+
+The easiest way to downgrade is by using `xdowngrade` from the `xtools` package.
+
```
Comment (Vaelatern) [inline src/xbps/packages/downgrading.md]: Please make a clear separation, making it clear this is the end of this subsection about `xdowngrade`. Maybe there is a bit of usage you can duplicate into here too?

Comment (flexibeast) [reply src/xbps/packages/downgrading.md]: Thanks for the feedback - PR updated.

File: src/xbps/packages/downgrading.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+The easiest way to downgrade is to use `xdowngrade` from the `xtools` package,
+specifying the package version you wish to downgrade to:
+
+```
+# xdowngrade /var/cache/xbps/pkg-1.0_1.xbps
+```
+
+## Via XBPS
+
+To downgrade using just XBPS, you must first add the desired package version to
```
Comment (bobertlo) [inline src/xbps/packages/downgrading.md]: To be clear, this is adding a locally cached older version of the package which is no longer contained in the repository index, correct? (or could be used if you got an older version somewhere else too.)

**Comment by bobertlo** [comment]
I think that one point could use clarification for the reader. Otherwise LGTM

File: src/xbps/packages/downgrading.md (hunk 2 above)
Comment (flexibeast) [reply src/xbps/packages/downgrading.md]: Thanks for the feedback - PR updated.

**Pull request merged**